<?xml version="1.0"?>
<launch>
    <arg name="keyboard" default="false" />

    <rosparam command="load" file="$(find volta_teleoperator)/config/joy.yaml" unless="$(arg keyboard)" />

    <node pkg="joy" type="joy_node" name="joy_node"  unless="$(arg keyboard)" />

    <node pkg="volta_teleoperator" type="volta_joy_teleop_node" name="joy_teleop" output="screen" unless="$(arg keyboard)">
    </node>

    <node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" name="teleop_twist_keyboard" if="$(arg keyboard)">
        <remap from="/cmd_vel" to="keyboard/cmd_vel"/>
    </node>

</launch>
